docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -0,0 +1,135 @@
+
+## Support for Azure credentials through configuration
+
+The [`Microsoft.Extensions.Azure`](https://www.nuget.org/packages/Microsoft.Extensions.Azure) library can automatically provide Azure service clients with a `TokenCredential` class by searching _appsettings.json_ or other configuration files for credential values using the default `IConfiguration` service for .NET. This approach allows developers to explicitly set credential values across different environments through configuration rather than through app code directly.
+
+The following credential types are supported via configuration:
+
+* [ClientCertificateCredential](#create-a-clientcertificatecredential-type)
+* [ClientSecretCredential](#create-a-clientsecretcredential-type)
+* [DefaultAzureCredential](#create-a-defaultazurecredential-type)
+* [ManagedIdentityCredential](#create-a-managedidentitycredential-type)
+* [WorkloadIdentityCredential](#create-a-workloadidentitycredential-type)
+
+## Configure Azure credentials
+
+Azure service clients registered with the `AddAzureClients` method are automatically configured with an instance of `DefaultAzureCredential`. You can override the global `DefaultAzureCredential` using credential values from configuration files when registering a client:
+
+```csharp
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    // Register BlobServiceClient using credentials from appsettings.json
+    clientBuilder.AddBlobServiceClient(builder.Configuration.GetSection("Storage"));
+
+    // Register ServiceBusClient using the fallback DefaultAzureCredential credentials
+    clientBuilder.AddServiceBusClientWithNamespace(
+        "<your_namespace>.servicebus.windows.net");
+});
+```
+
+The associated _appsettings.json_ file:
+
+```json
+"Storage": {
+    "serviceUri": "<service_uri>",
+    "credential": "managedidentity",
+    "clientid":  "<clientId>"
+}
+```
+
+### Create a `ManagedIdentityCredential` type
+
+You can create both user-assigned and system-assigned managed identities using configuration values. Add the following configuration values to your _appsettings.json_ file to create an <xref:Azure.Identity.ManagedIdentityCredential?displayProperty=fullName>.
+
+#### User-assigned identities
+
+1. Authenticate using a client ID:
+
+    ```json
+    {
+        "credential": "managedidentity",
+        "clientId":  "<clientId>"
+    }
+    ```
+
+1. Authenticate using a resource ID:
+
+    ```json
+    {
+        "credential": "managedidentity",
+        "resourceId":  "<managedIdentityResourceId>"
+    }
+    ```
+
+    The resource ID takes the form `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}`.
+
+#### System-assigned identities
+
+1. Authenticate using a client ID:
+
+    ```json
```

Comment:
It seems like we are missing the credential heading like we have in all of the other examples, e.g. https://github.com/dotnet/docs/pull/39164/files#diff-0a55f75251b3928717aa7ca16536dd78c01b389f5409a74da159a5efd279d39aR88